daml-foundations/daml-ghc/language-server/src/DA/Service/Daml/LanguageServer.hs

```diff
@@ -41,6 +41,9 @@ import Language.Haskell.LSP.Core (LspFuncs(..))
 import Language.Haskell.LSP.Messages
 import Language.Haskell.LSP.VFS
 
+tShow :: Show a => a -> T.Text
```

Comment:
`textShow` might be a more helpful name?

Reply:
Changed

Reply:
Thanks.